metaflow/plugins/aws/step_functions/step_functions.py

```diff
@@ -476,6 +476,8 @@ def _process_parameters(self):
         has_schedule = self._cron() is not None
         seen = set()
         for var, param in self.flow._get_parameters():
+            if param.IS_FLOW_PARAMETER:
```

Comment:
@saikonen can you add a comment here for why we continue? would be helpful later on